docs/pullFromDockerHub.md

```diff
@@ -0,0 +1,43 @@
+# Pull Rally Probot and run image from DockerHub
+The **GitHub Rally** Probot can be pulled from **DockerHub** and deployed using your container environment.
```

Comment:
```suggestion
The **Rally + GitHub** app can be pulled from **DockerHub** and deployed using your container environment.
```

Reply:
If I'm not mistaken, it should be _Docker Hub_ instead of _DockerHub_